modules/test/playwright/tests/frontend-editor-ckeditor-web/main/tests/ckeditor5/react_cet.spec.ts

```diff
@@ -22,37 +25,34 @@ export const test = mergeTests(
 	loginTest()
 );
 
-test.beforeEach(async ({ckeditorSamplePage, site}) => {
+test.beforeEach(async ({ckeditorSamplePage, page, site}) => {
 	await ckeditorSamplePage.createAndGotoSitePage({site});
 
 	await ckeditorSamplePage.selectTab('CKEditor 5');
 	await ckeditorSamplePage.selectTab('React + CET');
+
+	await waitForEditor({page});
 });
 
 test(
 	'Assert editor is rendered with features based on default configuration and Client Extension additions',
 	{tag: '@LPD-11235'},
-	async ({page}) => {
+	async ({classicPage, page}) => {
 		await expect(
 			page.getByText('Lorem ipsum dolor sit amet')
 		).toBeVisible();
 
-		const editorToolbar = page.getByLabel('Editor toolbar');
-
-		await expect(editorToolbar).toBeVisible();
-
 		const expectedButtons = [
 			'Accessibility help',
 			'Undo',
 			'Redo',
 			'A dropdown with a custom icon',
+			'Text alignment',
```

Comment:
Fix on an unrelated test failure, caused by external change in our workspace sample.